imap/index.c

```diff
@@ -5274,11 +5275,6 @@ static int extract_vcardbuf(struct buf *raw, charset_t charset, int encoding,
 
 #define IDLE_TIMEOUT (5 * 60)  /* 5 min */
 
-struct backend_ctx {
-    char *path;
-    int timeout;
-};
-
```

Comment:
and then we don't need a backend context at all, because we already have the path in the extractor context - and I looked at the timeout logic and it seemed un-necessary.